docs/user-guide.md

```diff
@@ -112,42 +120,3 @@ Then run the following to connect:
 cqlsh `docker-machine ip default` --cqlversion=3.2.0
 ```
 
-## Zipkin's internal storage structure (in Cassandra)
```

Comment:
sorry.. @clehene can confirm I was too tired to do reviews yesterday. This is indeed red, which means you are deleting it! Your above link to schema is great.